README.md

```diff
@@ -1,2 +1,14 @@
 # pattern-library
 A multi-language library containing implementations of common software design and architecture patterns 
+
+
+
+## Table of Contents
+
+
+
+## Pull Requests 
```

Comment:
I don't understand the purpose of this section - can you explain? My assumption is that you intend to include a list of PRs here? But we can track this through the standard GitHub tools (i.e. the [PR tab](https://github.com/nic-dgl104-winter-2024/pattern-library/pull)).

Did you have something else in mind for this section?

Reply:
For the Pull Requests section, I think we can encourage people to create pull requests and maybe some instruction for contributors to follow?

Reply:
For example , the content for this section could be 
" We welcome pull requests! Please follow the [Pull Request Template](.github/PULL_REQUEST_TEMPLATE.md) "

Reply:
I see. So a section to encourage contributions. **Let's call it `Contributions` instead,** since that's a bit more inclusive of what contributions might look like.

If this section grows we can consider breaking it out into a separate `CONTRIBUTING.md` document.